lib/workers/bin/evm_server.rb

```diff
@@ -1,5 +1,5 @@
-require "workers/evm_server"
-if MiqEnvironment::Process.is_rails_runner?
+if defined?(Vmdb::Application) && Vmdb::Application.initialized?
+  require "workers/evm_server"
```

Comment:
Replaced MiqEnvironment::Process call with what we really want, if you ran this with ruby by accident, the Vmdb::Application wouldn't be loaded or initialized.